syne_tune/optimizer/schedulers/searchers/regularized_evolution.py

```diff
@@ -0,0 +1,138 @@
+import copy
+
+from collections import deque
+from typing import Optional, Dict, List
+from dataclasses import dataclass
+
+from syne_tune.optimizer.schedulers.searchers import SearcherWithRandomSeed
+from syne_tune.search_space import Domain, Categorical, Float, Integer
+
+
+@dataclass
+class PopulationElement:
+    score: int = 0
+    config: dict = None
+
+
+class RegularizedEvolution(SearcherWithRandomSeed):
+    def __init__(
+            self,
+            configspace,
+            metric: str,
+            mode: str = 'min',
+            population_size: int = 100,
```

Comment:
Is it correct to say that this one needs 100 trials to be finished to the end before it is different from random search? Wow.